code/modules/mob/living/carbon/human/life.dm

```diff
@@ -339,7 +339,7 @@
 	// Tissues die without blood circulation
 	adjustBruteLoss(2)
 
-/mob/living/carbon/human/proc/handle_vamp_dreams()
+/*/mob/living/carbon/human/proc/handle_vamp_dreams()
```

Comment:
This proc is needed and necessary to remove the vampire's equivalent of sleepytime debuff.

Reply:
nvm disregard this